### PR TITLE
[MU3] fix #311621: fixed export audio on Windows

### PR DESCRIPTION
**Pull request by igorkorsukov**
I fixed this issue by another method https://github.com/musescore/MuseScore/pull/6531
But not working with MSVC build.

Possible, need use _get_osfhandle (https://stackoverflow.com/questions/50191480/how-to-get-a-windows-handle-from-a-qfile-or-qsavefile-for-reopenfile)
But I had the app crash.

So, now I implemented this solution - working (tested on MSVC build)

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
Probably should mention [#311621](https://musescore.org/en/node/311621)

File: mscore/exportaudio.cpp [hunk 1]
```diff
@@ -258,23 +263,33 @@ bool MuseScore::saveAudio(Score* score, const QString& name)
...
+                  wchar_t wpath[SF_FILENAME_LEN];
+                  int dwRet = MultiByteToWideChar(CP_UTF8, MB_ERR_INVALID_CHARS, path.constData(), -1, wpath, SF_FILENAME_LEN);
+                  if (dwRet == 0) {
+                        qCritical() << Q_FUNC_INFO << "filed get path: " << GetLastError() << "\n";  
+                        return false; 
                         }
-                  sf = sf_open_fd(file.handle(), SFM_WRITE, &info, false);
+                  sf = sf_wchar_open(wpath, SFM_WRITE, &info);
+#else  // Q_OS_WIN
+                  sf = sf_open(qPrintable(filename);
```
Comment (Jojo-Schmitz) [inline mscore/exportaudio.cpp:277]: closing paren missing!

Comment (igorkorsukov) [reply mscore/exportaudio.cpp]: > closing paren missing!

fixed

**Comment by igorkorsukov** [comment]
> Probably should mention [#311621](https://musescore.org/en/node/311621)

Thanks, added

**Comment by igorkorsukov** [comment]
> I've tried out the (Windows 64bit, the 32bit one seems pretty empty) artifact and it does solve the issue, it doeseacport to .flac, .ogg and .wav

Greate! Thank you! 
(32bit build disabled for pull requests, see steps https://github.com/musescore/MuseScore/pull/6665/checks?check_run_id=1247986069)

**Pull request merged**